viam-cartographer.go

```diff
@@ -281,6 +281,7 @@ func (slamSvc *builtIn) Position(ctx context.Context, name string, extra map[str
 		pInFrame = referenceframe.NewPoseInFrame(resp.GetComponentReference(), spatialmath.NewPoseFromProtobuf(resp.GetPose()))
 		returnedExt = resp.Extra.AsMap()
 	} else {
+		//nolint:staticcheck
```

Comment:
To confirm, we added these for the same reason we [currently exclude](https://github.com/viamrobotics/rdk/blob/6ea6876000cd6910e9f16ca994c8b9bd70f7e215/etc/.golangci.yaml#L100-L104) staticcheck from rdk's services/slam/ directory?

Will we automatically remove these lines at some point in the future, or should we make a note to remove these lines in RSDK-1053 or create a separate ticket for it?

Reply:
Great question. We need to manually remove these lines in the future. I created this ticket since I couldn't find an existing one: https://viam.atlassian.net/browse/RSDK-2339

Reply:
Perfect, thanks!